packages/playground/blueprints/src/lib/steps/activate-plugin.spec.ts

```diff
@@ -84,4 +138,61 @@ describe('Blueprint step activatePlugin()', () => {
 
 		expect(php.fileExists(createdFilePath)).toBe(true);
 	});
+
+	it('should activate a plugin if it redirects during activation', async () => {
+		const docroot = handler.documentRoot;
+		php.writeFile(
+			`${docroot}/wp-content/plugins/test-plugin.php`,
+			`<?php
+			/**
+			 * Plugin Name: Test Plugin
+			 */
+			add_action( 'activated_plugin', function( $plugin ) {
+				if( $plugin == plugin_basename( __FILE__ ) ) {
+					wp_redirect( admin_url( 'edit.php' ) );
+					exit();
+				}
+			} );
+			`
+		);
+		await expect(
+			activatePlugin(php, {
+				pluginPath: 'test-plugin.php',
+			})
+		).resolves.toBeUndefined();
+	});
+
+	it('should activate a plugin if it produces a output during activation', async () => {
+		const docroot = handler.documentRoot;
+		php.writeFile(
+			`${docroot}/wp-content/plugins/test-plugin.php`,
+			`<?php
+			/**
+			 * Plugin Name: Test Plugin
+			 */
+			echo 'Hello World';
+			`
+		);
+		await expect(
+			activatePlugin(php, {
+				pluginPath: 'test-plugin.php',
+			})
+		).resolves.toBeUndefined();
```

Comment:
The return values doesn't seem relevant here, what we're really after is `resolves.not.toThrow()`

Reply:
Done [c0dd231](https://github.com/WordPress/wordpress-playground/pull/2066/commits/c0dd231c3f84a7dceb4140aaab61646c0b7af44a)